es6/EntityFactory.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+import 'd2l-polymer-siren-behaviors/store/entity-store.js';
+
+// This creates and fetch a new entity. Whenever the entity changes onChange is called.
```

Comment:
May I suggest that we use something like [JSDocs](http://usejsdoc.org/) instead of raw comments?  Some editors can support jsdocs in their intellisense to give richer descriptions.

I think formating your comments like this is all you need
```suggestion
/** This creates and fetch a new entity. Whenever the entity changes onChange is called. */
```
Especially important if we're building an sdk for wider consumption.  Totally optional though

Reply:
You are completely correct here.